src/vm-status.js

```diff
@@ -3,15 +3,15 @@ export function canStart (state) {
 }
 
 export function canShutdown (state) {
-  return canRestart(state) || state === 'reboot_in_progress'
+  return canRestart(state) || state === 'reboot_in_progress' || state === 'paused'
 }
 
 export function canRestart (state) {
   return state && (state === 'up' || state === 'migrating')
 }
 
 export function canSuspend (state) {
-  return state && (state === 'up')
+  return state && (state === 'up' || state === 'paused')
```

Comment:
can we suspend (as in put to sleep) a VM that is paused in the emulator? (cc @michalskrivanek)